core-processor/src/executor.rs

```diff
@@ -437,8 +310,12 @@ where
         ActorTerminationReason::GasAllowanceExceeded => DispatchResultKind::GasAllowanceExceed,
     };
 
-    let page_update =
-        get_pages_to_be_updated::<E::Ext>(pages_initial_data, info.pages_data, static_pages);
+    // With lazy-pages we update some page data in storage,
+    // when it has been write accessed, so no need to compare old and new page data.
+    info.pages_data
```

Comment:
Please remove this redundant and incorrect log::trace. Better to make trace in journal notes handler.rs